app/src/Controller/ArticlesController.php

```diff
@@ -0,0 +1,117 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Controller;
+
+/**
+ * Articles Controller
+ *
+ * @method \App\Model\Entity\Article[]|\Cake\Datasource\ResultSetInterface paginate($object = null, array $settings = [])
+ */
+class ArticlesController extends AppController
+{
+    /**
+     * Initialize method.
+     *
+     * @return void
+     */
+    public function initialize(): void
+    {
+        parent::initialize();
+
+        $this->loadComponent('Paginator');
+        $this->loadComponent('Flash'); // Include the FlashComponent
+    }
+
+    /**
+     * Index method.
+     *
+     * @return \Cake\Http\Response|null|void Renders view
+     */
+    public function index()
+    {
+        $this->loadComponent('Paginator');
```

Comment:
Is it OK to load the component Paginator twice?

Reply:
The other one had been loaded through `initialize` method.

Reply:
sorry my bad, the paginator must be loaded only on initialize method